docs/resources/Guild.md

```diff
@@ -150,6 +150,7 @@ Guilds in Discord represent an isolated collection of users and channels, and ar
 | VERIFIED                         | guild is verified                                                                                                   |
 | VIP_REGIONS                      | guild has access to set 384kbps bitrate in voice (previously VIP voice servers)                                     |
 | WELCOME_SCREEN_ENABLED           | guild has enabled the welcome screen                                                                                |
+| CHANNEL_BANNER                   | guild has enabled the channel banner                                                                                |
```

Comment:
This is not correct. The `CHANNEL_BANNER` feature will be added automatically to a server once it reaches server boost level 3 (and currently also needs to be part of the guild experiment, though this will obviously change once the feature becomes available beyond the experiment). This is not something that server owners can (currently) toggle on or off.

Reply:
True, it's that it has access to the channel banner.